signwriting_transcription/pose_to_signwriting/data/prepare_poses.py

```diff
@@ -96,7 +96,17 @@ def process(args):
     print(f"Create pose {name} dataset.")
 
     print("Fetching train split ...")
-    dataset = load_dataset(dataset_root)
+    dataset = load_dataset(dataset_root, dataset_root)
+    if data_segment:
+        segment_dataset = load_dataset(data_segment, dataset_root)
+        modified_segment_dataset = []
+        for instance in segment_dataset:
+            instance = list(instance)
+            instance[0] = f"seg_{instance[0]}"
+            instance[3] = 'train' if instance[3] == 'test' else instance[3]
```

Comment:
you are *training* on the test data? this is really bad. you should filter out the test data